### DISCUSSION TITLE
upward continuation unit

### DISCUSSION THREAD

**Discussion by Hojat-Shirmard** [fatiando/community#157] [General]
Hi dear Developers,

I’ve just tested the upward continuation on the Australia-wide magnetic data, and it appears that the unit might not be in meters. It seems to be in degrees, as applying values between 0.1 and 0.5 produced sensible results, but using 500 caused all signals to disappear. Could you confirm if this is indeed the case?

Thank you!
Hojat

**Comment by LL-Geo** [comment]
Hi Hojat,

One thing to keep in mind is that we need to use projected gravity/magnetic data (in cartesian system) to do the upward continuation. Because the whole calculation is based on Fast Fourier Transform, which needs a regular spacing grid. In this case, you need to project your data from lat/lon into easting/northing first, and then apply the upward continuaiton filter.

**Comment by Hojat-Shirmard** [comment]
Thank you for your prompt response! I applied the Australia Albers Equal Area projection (EPSG:3577), and it worked perfectly!

**Comment by LL-Geo** [comment]
Good to know it works :)
